lib/capabilities.js

```diff
@@ -15,21 +19,148 @@ var async = require('async');
+ * Linux (and soon OS X) on x86 and x64.
+ */
+
 var capabilities = {
   watchdog: {
-    name: 'smart profiling',
+    name: 'Smart profiling',
     checks: [
       {
         check: function(cb) {
+          if (license('agent:watchdog')) {
+            cb(true);
+          } else {
+            cb(false, 'Smart profiling license is missing or expired.');
+          }
+        }
+      },
+      {
+        check: function(cb) {
           if (agent().internal.supports.watchdog) {
             cb(true);
           } else {
             cb(false, 'Host platform does not support smart profiling.');
           }
         }
+      },
+      {
+        check: function(cb) {
+          var addon = require('strong-agent/lib/addon');
+          if (addon) {
+            cb(true);
+          } else {
+            cb(false, 'Strong-agent addon not compiled.');
+          }
+        }
       }
     ]
-  }
+  },
+  tracing: {
+    name: 'Tracing',
+    checks: [
+      {
+        check: function(cb) {
+          if (license('agent:tracing')) {
+            cb(true);
+          } else {
+            cb(false, 'Tracing license is missing or expired.');
+          }
+        }
+      },
+      {
+        check: function(cb) {
+          if (process.env.STRONGLOOP_TRACING) {
+            cb(true);
+          } else {
+            cb(false, 'Tracing is disabled.');
+          }
+        }
+      }
+    ]
+  },
+  metrics: {
+    name: 'Metrics',
+    checks: [
+      {
+        check: function(cb) {
+          if (license('agent:metrics')) {
+            cb(true);
+          } else {
+            cb(false, 'Metrics license is missing or expired.');
+          }
+        }
+      },
+      {
+        check: function(cb) {
+          if (agent().metrics) {
+            cb(true);
+          } else {
+            cb(false, 'Metrics is disabled.');
+          }
+        }
+      },
+      {
+        check: function(cb) {
+          var addon = require('strong-agent/lib/addon');
+          if (addon) {
+            cb(true);
+          } else {
+            cb(false, 'Strong-agent addon not compiled.');
+          }
+        }
+      }
+    ]
+  },
+  cpuprofile: {
+    name: 'CPU profiling',
+    checks: [
+      {
+        check: function(cb) {
+          var addon = require('strong-agent/lib/addon');
+          if (addon) {
+            cb(true);
+          } else {
+            cb(false, 'Strong-agent addon not compiled.');
+          }
+        }
+      }
+    ]
+  },
+  heapsnapshot: {
+    name: 'Heap snapshot',
+    checks: [
+      {
+        check: function(cb) {
+          var heapdump = require('heapdump');
+          if (heapdump) {
+            cb(true);
+          } else {
+            cb(false, 'Heap snapshot not compiled.');
+          }
+        }
+      }
+    ]
+  },
+  patch: {
+    name: 'patch',
+    checks: [
+      {
+        check: function(cb) {
+          if (agent().dyninst.metrics) {
```

Comment:
I believe this will evaluate to a truthy value every time, once `agent.start()` or `agent.profile()` has been called, but this appears to be the same check that is already being done in lib/targetctl.js...